modules/ztp-gitops-ztp-max-spoke-clusters.adoc

```diff
@@ -0,0 +1,98 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-preparing-the-hub-cluster.adoc]
+
+:_content-type: REFERENCE
+[id="ztp-gitops-ztp-max-spoke-clusters_{context}"]
+= Recommended hub cluster specifications and managed cluster limits for {ztp}
+
+With {ztp-first}, you can manage thousands of clusters in geographically dispersed regions and networks.
+The Red Hat Performance and Scale lab successfully created and managed 3500 virtual {sno} clusters with a reduced DU profile from a single {rh-rhacm-first} hub cluster in a lab environment.footnote:[link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.7/html/install/installing#single-node[Creating and managing {sno} clusters with {rh-rhacm}]]
+
+In real world situations, the scaling limits for the number of clusters that you can manage will vary depending on various factors affecting the hub cluster.
+For example:
+
+Hub cluster resources::
```

Comment:
Based on the mod doc guide, I wonder if these should be discrete headings https://redhat-documentation.github.io/modular-docs/#reference-module-guidelines

I have never seen a list like this before and didn't see it called out in the style guide. I could have missed it if it is in there.

Reply:
It's a description list. See here for more details: https://docs.asciidoctor.org/asciidoc/latest/lists/description/